src/bun.js/bindings/bindings.zig

```diff
@@ -188,6 +188,10 @@ pub const ZigString = extern struct {
         return strings.eqlLong(utf16_slice.slice(), latin1_slice.slice(), true);
     }
 
+    pub fn contains(this: ZigString, other: ZigString) bool {
+        return strings.contains(this.slice(), other.slice());
```

Comment:
Unfortunately it's not this simple 

this can be latin1 or utf16

other can also be latin1 or utf16 

if the two ZigString are not both latin1, this will segfault or produce invalid results

The options are:
- Treat both as JSValue and then add a binding for "does this string contain this other string". You'll want to look in WTF::String's header
- Check if either are utf16 or either are latin1 and do a comparison byte by byte (use strings.contains except for utf16, it must be utf16Aligned() because it's 2x the bytes), otherwise encode both to utf8 (.toSlice) and compare that. This is probably easiest to implement but will increase memory usage and have more overhead due to the string conversion  step & allocation